get_involved.md

```diff
@@ -17,6 +17,22 @@ list.
 participate in real-time. See the [triage process](https://docs.pulpproject.org/bugs-features.html#triage) for
 more details. You can also read through the [triage archives](https://logs.pulpproject.org/pulp-meeting/).
 
+## Community Forum
+
+At the moment, we are trialling our very own [Pulp Community Discourse](https://discourse.pulpproject.org).
+Come along and introduce yourself and your project, or describe an issue you're having.
+We'd love to hear from you. 
+
+## Chat to us
+
+The Pulp community's primary communication channel is Matrix.
+
+We have corresponding Libera.Chat rooms for every Matrix room, so if you use IRC exclusively, you can find us there too.
+
+* Join [**pulp** on Matrix](https://matrix.to/#/!HWvLQmBGVPfJfTQBAu:matrix.org?via=libera.chat&via=matrix.org&via=ctrl-c.liu.se) for User support.
+
+* Join [**#pulp-dev** on Matrix](https://matrix.to/#/!aVApiNMtnstWbwDcVU:matrix.org?via=libera.chat&via=matrix.org&via=ctrl-c.liu.se) for Developer discussion.
```

Comment:
```suggestion
* Join [**pulp** on Matrix](https://matrix.to/#/!HWvLQmBGVPfJfTQBAu:matrix.org) for User support.

* Join [**#pulp-dev** on Matrix](https://matrix.to/#/!aVApiNMtnstWbwDcVU:matrix.org) for Developer discussion.
```

Reply:
Would you believe I went to great efforts to secure https://matrix.to/#/#pulp-dev:matrix.org and https://matrix.to/#/#pulp:matrix.org and didn't even use them here! 
Thanks. Have updated.